lib/ReactViews/FeatureInfo/generateCesiumInfoHTMLFromProperties.ts

```diff
@@ -0,0 +1,85 @@
+import JulianDate from "terriajs-cesium/Source/Core/JulianDate";
+import PropertyBag from "terriajs-cesium/Source/DataSources/PropertyBag";
+import isDefined from "../../Core/isDefined";
+
+const simpleStyleIdentifiers = [
+  "title",
+  "description",
+  "marker-size",
+  "marker-symbol",
+  "marker-color",
+  "stroke",
+  "stroke-opacity",
+  "stroke-width",
+  "fill",
+  "fill-opacity"
+];
+
+/**
+ * A way to produce a description if properties are available but no template is given.
+ * Derived from Cesium's geoJsonDataSource, but made to work with possibly time-varying properties.
+ */
+export function generateCesiumInfoHTMLFromProperties(
```

Comment:
Just thinking out loud, could we use react for rendering this using render method. It might improve readability

Reply:
I agree - but I don't think I have time to do this right now unfortunately